frontends/PyCDE/test/polynomial.py

```diff
@@ -71,6 +73,7 @@ def build(self, top):
     x = hw.ConstantOp.create(i32, 23)
     poly = PolynomialCompute("example", [62, 42, 6], x=x)
     PolynomialCompute("example2", [62, 42, 6], x=poly.y)
+    PolynomialCompute("example2", [1, 2, 3, 4, 5], x=poly.y)
```

Comment:
Add a new `CHECK` which tests this functionality?

Reply:
And I didn't realize that the test was broken. This is what happens when meetings only allow me to work 15 to 30 mins at a time.